electron/main.js

```diff
@@ -144,6 +144,8 @@ function showMainWindow() {
     },
   });
 
+  let unread_count = 0;
```

Comment:
now I see it.. but I think we are putting too much logic on something that is quite minor..